index.js

```diff
@@ -110,7 +110,10 @@ module.exports = (app, options) => {
         })
 
         const contentType = (res.headers['content-type'] || res.headers['Content-Type'] || '').split(';')[0]
-        const isBase64Encoded = options.binaryMimeTypes.indexOf(contentType) > -1
+        const contentEncoding = res.headers['content-encoding'] || res.headers['Content-Encoding']
```

Comment:
if header is set with `reply.header()` function it will always be lowercase. I think that's an issue in the library here for the rest of the codebase.

See: https://github.com/fastify/fastify/blob/5a785ee49b2119d66cc08597d4f8d2f9cb506ab1/lib/reply.js#L223

Reply:
it's ok to check for both cases I think... if someone is not using the header function

Reply:
Oh I see. No problem, changing to lowercase only.

Reply:
That means they will setting the header at the `raw` level.

Reply:
Yes, this may happen

Reply:
(didn't see @adrai response) So, better to leave both options, lowercase and camelcase?

Reply:
imo yes